lib/ansible/module_utils/basic.py

```diff
@@ -845,9 +846,6 @@ def __init__(self, argument_spec, bypass_checks=False, no_log=False,
         self._legal_inputs = ['_ansible_%s' % k for k in PASS_VARS]
         self._options_context = list()
 
-        # set tempdir to remote tmp
-        self.tempdir = os.environ.get('ANSIBLE_REMOTE_TEMP', None)
```

Comment:
Shouldn't we still preserve the set in `__init__`, just using the value that came over in _ansible_tempdir (ie moved to after load_params)? Modules could theoretically be using this (unless it just got added in 2.5 devel...)

Reply:
NM, data-driven set later in _check_arguments